modules/test/playwright/tests/object-web/listTypeDefinitions.spec.ts

```diff
@@ -429,7 +429,9 @@ test.describe('ensure picklist translation', () => {
 
 		// Preview form
 
-		await page.waitForTimeout(200);
+		await page.waitForResponse(
+			'http://localhost:8080/o/dynamic-data-mapping-form-context-provider/'
```

Comment:
Same as what I commented in [evaluation.es.js](https://github.com/liferay-bpm/liferay-portal/pull/4401/files#diff-76a61373a4219750d7b74674f386ce953e47437d0333c0a2561b777dbc292f4d)